templates/slurm.conf.j2

```diff
@@ -104,33 +104,32 @@ AccountingStorageType=accounting_storage/filetxt
 Epilog=/etc/slurm/slurm.epilog.clean
 {% for part in openhpc_slurm_partitions %}
 {% for group in part.get('groups', [part]) %}
-{% if group.num_nodes > 1 %}
-NodeName={{group.cluster_name|default(openhpc_cluster_name)}}-{{group.name}}-[0-{{group.num_nodes|int-1}}] \
-{% else %}
-NodeName={{group.cluster_name|default(openhpc_cluster_name)}}-{{group.name}}-0 \
-{% endif %}
-{% if 'ram_mb' in group %}
-    RealMemory={{group.ram_mb}} \
-{% endif %}
 {% set group_name = group.cluster_name|default(openhpc_cluster_name) ~ '_' ~ group.name %}
 {# If using --limit, the first host in each group may not have facts available. Find one that does. #}
 {% set group_hosts = groups[group_name] | intersect(play_hosts) %}
-{% if group_hosts | length > 0 %}
+
+NodeName=DEFAULT State=UNKNOWN {% if 'ram_mb' in group %} RealMemory={{group.ram_mb}}{% endif %}{% if group_hosts | length > 0 %}\
```

Comment:
Because NodeName=DEFAULT is additive (AFAICT from docs), if one group has ram_mb set and the next does not, it will pick up the previous group's value.

Reply:
Ah good catch. Looks like I need to set it to slurm's default value of =1 if not defined

Reply:
Fixed by a97118f